source/NVDAObjects/behaviors.py

```diff
@@ -365,6 +365,38 @@ def event_gainFocus(self):
 	def event_loseFocus(self):
 		self.stopMonitoring()
 
+class LiveTextHistoryMixin(object):

```

Comment:
SHouldn't this inherrit from LiveText
I also think that you can leave out the word Mixin from the class name, as behaviors contains all mixins.

Reply:
I would, but wouldn't that cause issues (`liveText` would be in the MRO twice, once as the parent of the mixin and once as a parent of the object itself)?

Reply:
I guess the dynamic class logic in NVDAObjets will deal with this, but it needs to be tried out.